internal/config/config.go

```diff
@@ -64,10 +67,12 @@ type API struct {
 }
 
 type EKS struct {
-	AccountID   string        `mapstructure:"account_id"`
-	Region      string        `mapstructure:"region"`
-	ClusterName string        `mapstructure:"cluster_name"`
-	APITimeout  time.Duration `mapstructure:"api_timeout"`
+	AccountID                         string             `mapstructure:"account_id"`
+	Region                            string             `mapstructure:"region"`
+	ClusterName                       string             `mapstructure:"cluster_name"`
+	APITimeout                        time.Duration      `mapstructure:"api_timeout"`
+	APINodeLifecycleDiscoveryDisabled *bool              `mapstructure:"api_node_lifecycle_discovery_disabled"`
```

Comment:
This is more of a question: I wonder what benefits to use `*bool` here instead of `bool` with default value? Is to emphasise that this param is optional?

Reply:
No this was my mistake. At the beginning I called the variable Enabled and it required this to be a pointer otherwise we'd have issues initializing it. After changing it to Disabled I didn't change it to bool